src/OrchardCore.Modules/OrchardCore.Flows/ViewModels/BagPartEditViewModel.cs

```diff
@@ -16,6 +16,8 @@ public class BagPartEditViewModel
         [BindNever]
         public BagPart BagPart { get; set; }
 
+        [System.Text.Json.Serialization.JsonIgnore]
```

Comment:
Isn't there a common attribute that doesn't require referencing these libraries?

Reply:
I added as - I guess NewtosoftJson's  `JsonIgnore` wont work for `System.Text.Json`

Reply:
`[IgnoreDataMember]`, or `[NonSerialized]` might work for both libraries

Reply:
`NonSerialized is not allowed on properties
However `IgnoreDataMember` is working - Updated the PR